### PR TITLE
fix travis-ci build error due to bash quotes issue

### PR DESCRIPTION
**Pull request by ahalim-intel**
travis-ci build build is failing after merging #86.
The deployment script contains both single and double quotes and
causing issues.

### CODE REVIEW

**Comment by zshi-redhat** [comment]
@ahalim-intel can you see the log from previous travis run: https://travis-ci.org/github/intel/sriov-network-device-plugin/builds/663967719?utm_medium=notification&utm_source=github_status ?
I tried to see the error message, but didn't see anything there.

**Comment by zshi-redhat** [comment]
btw, it seems the travis is in queue for a long time before executing.

**Comment by ahalim-intel** [comment]
> @ahalim-intel can you see the log from previous travis run: https://travis-ci.org/github/intel/sriov-network-device-plugin/builds/663967719?utm_medium=notification&utm_source=github_status ?
> I tried to see the error message, but didn't see anything there.

is this what you were looking for?
https://travis-ci.org/github/intel/sriov-network-device-plugin/builds/661915886?utm_medium=notification&utm_source=github_status

**Comment by ahalim-intel** [comment]
> btw, it seems the travis is in queue for a long time before executing.

Seems like the queue time is very long! There are too many jobs.

**Comment by ahalim-intel** [comment]
By the way, Even if the travis-ci shows as build success, it won't confirm that this will fix the issue encountered in deploy stage. The deploy is triggered on merge only, not on PRs.

**Comment by zshi-redhat** [comment]
> By the way, Even if the travis-ci shows as build success, it won't confirm that this will fix the issue encountered in deploy stage. The deploy is triggered on merge only, not on PRs.

Yeah, I was trying to find the failed traivs build after merging the multi-arch patch and verify this PR fix the issue.

**Comment by ahalim-intel** [comment]
Build is failing in travis-ci for ppc64le with following errors:

```
Step 7/12 : RUN apk add hwdata-pci
 ---> Running in 533cbd7926b6
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/ppc64le/APKINDEX.tar.gz
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
```

Looks like the `hwdata-pci` package for alpine:ppc64le is not available?
@Mmduh-483 can you please take a look?

File: .travis.yml [hunk 1]
```diff
@@ -29,27 +29,23 @@ deploy:
...
+        docker push nfvpe/sriov-device-plugin:$TRAVIS_TAG;
       fi
-      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
-      docker push nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
-      echo done'
+      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$TRAVIS_TAG-$ARCH;
+      docker push nfvpe/sriov-device-plugin:$TRAVIS_TAG-$ARCH;
     on:
       tags: true
       all_branches: true
```
Comment (Mmduh-483) [inline .travis.yml]: @ahalim-intel I think you can test if you setup a Travis CI on your forked repo, this line show it should run the deploy section if the PR is pushed on any branch.

**Comment by Mmduh-483** [comment]
> @Mmduh-483 can you please take a look?

@ahalim-intel I faced this issue before in the my PR, it was solved after adding line 16 `- sudo systemctl restart docker`

**Comment by Mmduh-483** [comment]
@ahalim-intel I see it didn't fail on building the image in merge [CI](https://travis-ci.org/github/intel/sriov-network-device-plugin/jobs/663111323?utm_medium=notification&utm_source=github_status), maybe something stuck in the travis itself, can you please re-trigger it.

**Comment by ahalim-intel** [comment]
> @ahalim-intel I see it didn't fail on building the image in merge [CI](https://travis-ci.org/github/intel/sriov-network-device-plugin/jobs/663111323?utm_medium=notification&utm_source=github_status), maybe something stuck in the travis itself, can you please re-trigger it.

I've done that and it's still the same.

**Comment by moshe010** [comment]
It seem that hw-pci is not exist in http://dl-cdn.alpinelinux.org/alpine/v3.11/main/ppc64le/
can you update the Dockerfile of ppc with hwids-pci?

**Comment by moshe010** [comment]
I mean hwdata-pci not exist

**Comment by ahalim-intel** [comment]
> It seem that hw-pci is not exist in http://dl-cdn.alpinelinux.org/alpine/v3.11/main/ppc64le/
> can you update the Dockerfile of ppc with hwids-pci?

Yes, this seems to be the issue here. If that package is not available from apk are there any other alternatives?

If we can't find alternatives then we may need to disable the ppc64le image build for time being until a solution is found. We cannot leave the travis-ci not updating the default image on new merge.

cc @zshi-redhat

**Comment by zshi-redhat** [comment]
> > It seem that hw-pci is not exist in http://dl-cdn.alpinelinux.org/alpine/v3.11/main/ppc64le/
> > can you update the Dockerfile of ppc with hwids-pci?
> 
> Yes, this seems to be the issue here. If that package is not available from apk are there any other alternatives?

I agree, we need to find alternative for hwdata-pci if exist.
> 
> If we can't find alternatives then we may need to disable the ppc64le image build for time being until a solution is found. We cannot leave the travis-ci not updating the default image on new merge.
> 

I think we can disable ppc64le until above pkg issue is fixed.

> cc @zshi-redhat

commenting out ppc64le in build matrix looks good to me.

**Comment by ahalim-intel** [comment]
@zshi-redhat This build seems to be OK. Are we happy to merge this?

**Comment by zshi-redhat** [comment]
> @zshi-redhat This build seems to be OK. Are we happy to merge this?

@ahalim-intel yes, it looks good to me.

**Pull request merged**